src/component/utils/getDefaultKeyBinding.js

```diff
@@ -85,6 +85,8 @@ function getDefaultKeyBinding(
       return hasCommandModifier(e) ? 'code' : null;
     case 75: // K
       return (!isWindows && isCtrlKeyCommand(e)) ? 'secondary-cut' : null;
+    case 77: // M
+      return hasCommandModifier(e) ? 'split-block' : null;
```

Comment:
Yep thanks for catching that. I'll resubmit.

Is there any test code for key commands? I was thinking of adding a test around the button press but didn't see any good examples to work off of.